jumpstart-foundation-models/text2text-generation-flan-t5-xl/text2text-generation-flan-t5-xl.ipynb

```diff
@@ -0,0 +1,1455 @@
+{
```

Comment:
<p>This all is not needed, if we only demonstrate pre-packaged models, correct? </p><p> </p><p>My suggestion is: let's remove this, use this notebook only for the pre-packaged models, and soon we will pre-pack all text2text generation models, and then this will work for all the models.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3795/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>